src/Categories/Diagram/Duality.agda

```diff
@@ -179,3 +180,50 @@ module _ {F : Bifunctor (Category.op D) D C} where
     ; unique    = unique
     }
     where open Coend.Coend e
+
+
+module DualityConversionProperties where
+  private
+    Coequalizer⇔coEqualizer : ∀ (coequalizer : Coequalizer f g) →
+      coEqualizer⇒Coequalizer (Coequalizer⇒coEqualizer coequalizer)≡ coequalizer
+    Coequalizer⇔coEqualizer _ = refl
+
+
+    coPullback⇔Pushout : ∀ (coPullback : Pullback f g) →
+      Pushout⇒coPullback (coPullback⇒Pushout coPullback) ≡ coPullback
+    coPullback⇔Pushout _ = refl
+
+    module _ {F : Functor J C} where
+      open Functor F renaming (op to Fop)
+
+      private
```

Comment:
I think this module and the next one live in the private above right? so there isn't a need to add the private here.

Reply:
done